Dockerfile

```diff
@@ -77,4 +77,7 @@ ONBUILD COPY src/Pyz $WORKDIR/src/Pyz
 ONBUILD COPY config $WORKDIR/config
 ONBUILD COPY public $WORKDIR/public
 ONBUILD RUN /entrypoint.sh build-shop
+ONBUILD COPY codeception.yml $WORKDIR/
+ONBUILD COPY setup_test $WORKDIR/
```

Comment:
like in the `.dockerignore` file, I would prefere `ONBUILD COPY *test* $WORKDIR/` here